frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -112,8 +112,7 @@ common_settings.screen = {
         require("ui/elements/screen_eink_opt_menu_table"),
         require("ui/elements/menu_activate"),
         require("ui/elements/screen_disable_double_tap_table"),
-        require("ui/elements/flash_ui"),
-        require("ui/elements/flash_keyboard"),
+        require("ui/elements/screen_toggle_gsensor"),
```

Comment:
I think this sensor menu item should only be shown on device that has one :)
Could be done just like it's done just below for color screen and full screen on android.

Reply:
Agreed, that was already on my TODO list ;).